src/mpbconf.h.in

```diff
@@ -13,4 +13,5 @@
 #include "mpb/eigensolver.h"
 #include "mpb/maxwell.h"
 
+extern void (*mpb_printf_callback)(const char *s);
```

Comment:
Should be in mpi_utils.h, I think?

Reply:
I would agree, but `mpi_utils.h` doesn't get installed by `make install`. The only headers available in the installed library (the only headers Meep will see) are
```
mpb.h
mpb/
  eigensolver.h
  matrices.h
  maxwell.h
  scalar.h
```